UltraStar Play/Assets/Common/Audio/Recording/MicrophonePitchTracker.cs

```diff
@@ -9,8 +9,15 @@ public class MicrophonePitchTracker : MonoBehaviour
     // TODO: use 44100Hz (if supported) or fallback to default microphone sample rate
     private const int SampleRateHz = 44100;
 
+    // Longest period of singable notes (C2) requires 674 samples at 44100 Hz sample rate.
+    // Thus, 1024 samples should be sufficient.
+    private const int MaxSampleCountToUse = 1024;
```

Comment:
this should probably not have a maximum, so that setups with higher sample rates still work well. Also, the more samples, the more resilient the pitch detection should be against noise (especially from speakers). The pitch detection code is (or was) so fast, that even in a raspberry pi it was able to to pitch detection for 6 microphones without influencing frame rate. (only in debug-mode it was slow because of missing optimizations)

Reply:
I am unsure. I would leave it for now, so you can play with this parameter (or remove it) once the other PRs are done.